app/models/filters.rb

```diff
@@ -0,0 +1,29 @@
+module Filters
+  def self.research_and_statistics_filters
+    [
+      {
+        'key' => 'upcoming_statistics',
+        'label' => 'Statistics (upcoming)',
+        'filter' => {
+          'release_timestamp' => "from:#{Date.today.iso8601}",
+          'format' => %w(statistics_announcement)
```

Comment:
There's an argument for moving more of the data in the finder content item into this app.
The finders are completely tied to the finder-frontend - stats finder is a good example.
There's not a good argument for putting this stuff into rummager.
Perhaps we need an application in between.. `finder-api`? 😃